### PR TITLE
Feature#5 initialize project

### LINKED ISSUE #5: Initialize Project
# REQUIREMENTS
- Initialize rails project with postgresql
- Add bootstrap

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implemented/Why did you implemented this:
 - The project was created using Ruby on Rails.
 - Installed the dependencies bootstrap and popper.js.
 - Project created with PostgreSQL as main database.
 - Generated the pages_controller and a new page home, changing the main route to this one.

### CODE REVIEW

**Review by kevinnio — DISMISSED** [review]
Nice.

I know this kata uses Rails 6, but Rails 7 is already out. Maybe you guys would like to try setting up a Rails 7 app as an optional exercise just to see if it's any different.

**Review by xFerchoVJ — DISMISSED** [review]
LGTM!

**Comment by DerekAyala** [comment]
@YilKanda can you give us a review ?

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline app/assets/stylesheets/pages.scss]: If you don't occupy this file, you can delete it.

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
+end
```
Comment (YilKanda) [inline app/helpers/pages_helper.rb]: If you don't occupy this file, you can delete it.

File: app/javascript/stylesheets/application.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import 'bootstrap'
```
Comment (YilKanda) [inline app/javascript/stylesheets/application.scss]: Add a blank line.

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+#     url: <%= ENV['MY_APP_DATABASE_URL'] %>
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
+production:
+  <<: *default
+  database: kata_07_reloj_checador_k7_n21_t3_production
+  username: kata_07_reloj_checador_k7_n21_t3
+  password: <%= ENV['KATA_07_RELOJ_CHECADOR_K7_N21_T3_DATABASE_PASSWORD'] %>
```
Comment (YilKanda) [inline config/database.yml]: Do not upload this file to GitHub, instead you can put an example file: database.yml.example, with the basic configuration to make your project work. This file places it in the .gitignore.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 4 above)
Comment (DerekAyala) [reply config/database.yml]: I added to the .gitignore, is necessary delete this file from the remote repository ?

Comment (DerekAyala) [reply config/database.yml]: or just untrack this file

**Comment by DerekAyala** [comment]
@YilKanda could you give us again a review?

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by Villaedi — APPROVED** [review]

**Pull request merged**